src/components/Badge/Badge.stories.tsx

```diff
@@ -26,10 +27,8 @@ export default {
         onDismiss: { table: { disable: true } },
     },
     args: {
-        style: BadgeStyle.Primary,
         children: 'Text',
-        size: 'medium',
-        emphasis: BadgeEmphasis.Strong,
+        icon: <IconDocumentText />,
```

Comment:
I wanted it to be possible to preview the component with as few attributes set as possible, like a 'blank slate'. I noticed that it's not possible to set the `size` or `emphasis` to a null value in Storybook, which I needed to be able to test. So that's why I removed those attributes.

I also added an Icon as an example because it's tricky to generate the JSON for that, but I can remove it. What do you think?

Reply:
What JSON are you trying to generate 🤔 , I'm confused.
The stories and the cypress tests are not related so I think it would be better if you reset the Story file the way it was.
It's breaking the Status Only Story BTW as that IconDocument is always showing.

Reply:
<img width="871" alt="Screenshot 2023-01-17 at 11 12 14" src="https://user-images.githubusercontent.com/4213299/212870952-d3d9414f-247d-41d7-acd9-c96723d814b6.png">

I meant the JSON used to display the icon (shown in the screenshot).  That's the JSON that storybook generates if you add `<IconDocumentText />` to the `args` object. It's hard to preview what the badge would look like with an icon without adding it into the args.

I'll revert the story file changes though.